centml/sdk/auth.py

```diff
@@ -0,0 +1,61 @@
+import time
+import sys
+import os
+import json
+import requests
+import jwt
+
+from .config import Config
+
+
+def refresh_centml_token(refresh_token):
+    api_key = Config.firebase_api_key
+
+    cred = requests.post(
+        f"https://securetoken.googleapis.com/v1/token?key={api_key}",
+        headers={"content-type": "application/json; charset=UTF-8"},
+        data=json.dumps({"grantType": "refresh_token", "refreshToken": refresh_token}),
+        timeout=3,
+    ).json()
+
+    with open(Config.centml_cred_file, 'w') as f:
+        json.dump(cred, f)
+
+    return cred
+
+
+def store_centml_cred(token_file):
+    with open(token_file, 'r') as f:
```

Comment:
there are no checks to see if the file exists. Python will of course throw an error, but maybe we want to check fail gracefully with a "file does not exist" message